tests/Type/DefinitionTest.php

```diff
@@ -752,18 +752,69 @@ public function testInputObjectTypeAllowsRecursiveDefinitions(): void
 
     public function testAllowsInputTypeWhichDefinesItFieldsAsClosureReturningFieldDefinitionAsArray(): void
     {
-        $objType = new InputObjectType([
+        $fieldName = 'f';
+        $objType   = new InputObjectType([
             'name'   => 'SomeInputObject',
             'fields' => [
-                'f' => static fn (): array => [
+                $fieldName => static fn (): array => [
                     'type' => Type::string(),
                 ],
             ],
         ]);
 
         $objType->assertValid();
 
-        self::assertSame(Type::string(), $objType->getField('f')->getType());
+        self::assertSame(Type::string(), $objType->getField($fieldName)->getType());
+    }
+
+    public function testAllowsInputTypeWhichDefinesFieldsWithLazyTypes(): void
```

Comment:
Most input object tests are around line 1600, maybe to move these down there as well?

Reply:
I just checked and they are also here :/ starting with L716. I'd postpone it into another PR.

Reply:
I think I was able to put them in some kind of coherent order now 😅

Reply:
Right, sure. I got some ideas about testing, how to improve it. I'll write it down.